package.json

```diff
@@ -19,6 +19,9 @@
     "three": "^0.112.1",
```

Comment:
Iis this no longer needed then?
```suggestion
```

Reply:
peerDependencies are not included in the repo as dependencies.

Reply:
okay thanks!